src/PaperG/FirehoundBlob/BasicInfo.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace PaperG\FirehoundBlob;
+
+
+class BasicInfo implements BlobInterface
+{
+    const NAME = "name";
```

Comment:
Thinking more about this, should we add versioning to the blob? Or basic is so... basic that we don't need to version it at all?